meson.build

```diff
@@ -76,6 +76,10 @@ shared_c_args = [
     '-D_AST_no_spawnveg=1',
 ]
 
+if get_option('build-api-tests') == true
```

Comment:
Shall it be `coshell` instead of `build-api-tests` ?

Reply:
Yes, fixed